src/components/hv-date-field/index.js

```diff
@@ -237,7 +191,28 @@ export default class HvDateField extends PureComponent<
       props.maximumDate = maxDate;
     }
 
-    return <DatePickerIOS {...props} />;
+    return <DateTimePicker {...props} />;
+  };
+
+  /**
+   * Unlike iOS, the Android picker natively uses a modal. So we don't need
+   * to wrap it in an extra component, just render it when we want the modal
+   * to appear.
+   */
+  renderPickerModalAndroid = (): ?ReactNode => {
+    if (!this.state.focused) {
+      return null;
+    }
+    const onChange = (evt: Event, date: Date) => {
+      if (date === undefined) {
```

Comment:
it looks like `date` cannot be undefined from the flowtype definition. You might need to change the signature of this handler everywhere it's used to reflect this

Reply:
Good catch, updating to match the library:
https://github.com/react-native-datetimepicker/datetimepicker/blob/master/src/types.js#L50